src/main/java/frc/robot/auto/AutoRoutines.java

```diff
@@ -304,7 +307,34 @@ public Command getAutonomousCommand(int selectAuto) {
                     new ArmCommand(ArmSetpoint.HOME_AFTER_PICKUP).withTimeout(1)
                 ).andThen(new ChargeStationLock()));
 
-            // case 8:
+            case 8: // Id8 but id7 for driveteam
```

Comment:
Is it better to put this for case 1?
Then id1: bottom red and id2 for bottom blue

Reply:
sure, will change.

We did run id1 twice. Worked once and actually balanced. Failed another time.